kafka/vendor/selectors34.py

```diff
@@ -93,7 +93,8 @@ def __iter__(self):
         return iter(self._selector._fd_to_key)
 
 
-class BaseSelector(six.with_metaclass(ABCMeta)):
+@six.add_metaclass(ABCMeta)
```

Comment:
Pull request to vendored library: https://github.com/berkerpeksag/selectors34/pull/7